confirmed_malicious.rego

```diff
@@ -1,27 +1,31 @@
-package policy
+# METADATA
+# title: Confirmed Malicious
+# description: |
+#   Return a violation if the pacakge or author is tied to known malicious behavior
+package policy.v1
 
 import rego.v1
 
 # Returns a violation if the author is known malicious
-# METADATA
-# scope: rule
-# schemas:
-#   - data.issue: schema.issue
-issue contains "Author has published malicious packages" if {
-	data.issue.tag == "CA0001"
+deny contains issue if {
```

Comment:
Similar to the `# METADATA` comment at the top above `package policy.v1`, if you put a `# METADATA` block with a title above a rule, the title will be used in the rejection instead of the title of the policy.

eg

```
# METADATA
# title: Author has published malicious packages
deny contains issue if {
```

We can probably get rid of the comments above the rules if we put the titles in.

Reply:
Updated